src/quo2/components/reactions/resource.cljs

```diff
@@ -0,0 +1,12 @@
+(ns quo2.components.reactions.resource
+  (:require-macros [quo2.components.reactions.resource :refer [resolve-all-reactions]]))
+
+(def ^:private reactions
+  (resolve-all-reactions))
+
+(defn get-reaction
+  [reaction]
+  (assert (keyword? reaction) "Reaction should be a keyword")
+  (assert (= "reaction" (namespace reaction))
+          "Reaction keyword should be namespaced with :reaction")
```

Comment:
why not `:r/` ? 🤔 we have `i` for icon, `t` for translation already

Reply:
I thought about this @J-Son89, but icons are ubiquitous and often used statically, whereas reactions are mostly dynamic content, so we'll rarely use them as we use icons. Thus, IMO saving characters would hurt readability.